scene/gui/label.cpp

```diff
@@ -369,7 +370,10 @@ PackedStringArray Label::get_configuration_warnings() const {
 		int64_t glyph_count = TS->shaped_text_get_glyph_count(text_rid);
 		for (int64_t i = 0; i < glyph_count; i++) {
 			if (glyph[i].font_rid == RID()) {
-				warnings.push_back(RTR("The current font does not support rendering one or more characters used in this Label's text."));
+				Dictionary warning;
+				warning["message"] = RTR("The current font does not support rendering one or more characters used in this Label's text.");
+				warning["property"] = "text"; // REDMSER TODO: Try accessing the font using its theme override path, see if that actually shows the warning in the correct place.
```

Comment:
TODO before merge: If the two commits stay split, these changes to Label should be reverted, since they were just for testing.